### PR TITLE
Uaf 2277

### PR DESCRIPTION
**Pull request by jshard**
Put steps to validate this in jira ticket.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,4 +24,14 @@ ENV ORA_LIB_DIR=$ORA_BASE_DIR/instantclient_12_1
...
 
+#liquibase env variables
+ENV LIQUIBASE_HOME=/usr/share/liquibase
+
+# create liquibase home directory and subdirectories
+RUN mkdir $LIQUIBASE_HOME
+RUN mkdir $LIQUIBASE_HOME/lib
+
+# put liquibase and ojdbc jar files in lib dir
+COPY classes/liquibase-3.3.5.jar $LIQUIBASE_HOME/lib/
```
Comment (heather-lo) [inline Dockerfile:35]: Do you need to cd into a directory before copying the .jar file? I am not sure where the classes directory is.

Comment (jshard) [inline Dockerfile:35]: No, the path for the src file is relative to the source directory that is being built, or what they call the context of the build: https://docs.docker.com/engine/reference/builder/#copy
This is the same convention Mark used to copy over his files, and I also tested this in our UAF Jenkins.

Comment (jshard) [inline Dockerfile:35]: And to clarify further, this is the classes dir in the ua-eas/docker-jenkins repo.

**Comment by heather-lo** [comment]
Looks good.

**Pull request merged**